Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/SendConfirmationTaskModule/sendConfirmationTaskModule.tsx

```diff
@@ -69,8 +69,8 @@ class SendConfirmationTaskModule extends React.Component<RouteComponentProps, IS
             this.getItem(id).then(() => {
                 getConsentSummaries(id).then((response) => {
                     this.setState({
-                        teamNames: response.data.teamNames,
-                        rosterNames: response.data.rosterNames,
+                        teamNames: response.data.teamNames.sort(),
```

Comment:
Is it guaranteed that neither of these (teamNames and rosterNames) is going to be null/undefined?

Reply:
If there is no team or roster selected, api will return [ ].  So teamNames and rosterNames will not be null or undefined.